src/workloads/workloads.csproj

```diff
@@ -111,7 +111,9 @@
     <ItemGroup>
       <!-- Set SupportsMachineArch metadata to true if a specific manifest needs to support arm64 for Visual Studio. DO NOT set if for all feature bands as
            it changes the SWIX authoring and older versions of Visual Studio will fail to install. -->
-      <ManifestPackages Include="$(PackageSource)Microsoft.NET.Workload.Mono.ToolChain.*Manifest-*.nupkg" MsiVersion="$(MsiVersion)"/>
+      <ManifestPackages Include="$(PackageSource)Microsoft.NET.Workload.Mono.ToolChain.*Manifest-%(WorkloadSdkBandVersions.Identity).*.nupkg"
```

Comment:
Can we validate this change in the PR itself? Maybe with a one-off run of the pipeline for this?

Reply:
we're using the same pattern in emsdk and this was built in the maui arm branch previously